Products/ZenModel/UserSettings.py

```diff
@@ -449,6 +449,28 @@ def manage_deleteUsers(self, userids=(), REQUEST=None):
                 audit('UI.User.Delete', username=userid)
             return self.callZenScreen(REQUEST)
 
+    security.declareProtected(ZEN_MANAGE_DMD, 'manage_unlockUsers')
+    @validate_csrf_token
+    def manage_unlockUsers(self, userids=(), REQUEST=None):
+        """Unlocked a list of zenoss users from the system which are locked
+            due to numerous bad login attempts.
+        """
+
+        for user_id in userids:
+            try:
+                del self.acl_users.sessionAuthHelper.attempt[user_id]
+            except KeyError:
+                pass
```

Comment:
Prefer to use exceptions for 'big' errors and I think exceptions are expensive, even in Python.  So, there are two cleaner alternative ways to code this:
```
for user_id in userids:
    self.acl_users.sessionAuthHelper.attempt.pop(user_id, None)
```
or
```
helper = self.acl_users.sessionAuthHelper
for user_id in (uid for uid in userids if uid in helper.attempt):
    del helper.attempt[user_id]  # or helper.attempt.pop(user_id)
```